src/utils.ts

```diff
@@ -20,6 +20,22 @@ export function toArray<T>(arr: T | T[]): T[] {
   return [arr]
 }
 
+export function split(str: string, sep: string) {
+    return str == '' ? [] : str.split(sep)
+}
+
+export function join(arr: string[], sep: string) {
+	return arr == null ? '' : Array.prototype.join.call(arr, sep)
+}
```

Comment:
Do we need them?

Reply:
not necessarily, just added them for convience, made the code a little bit more readable